_plugins/removefrontmatter.rb

```diff
@@ -0,0 +1,23 @@
+module Jekyll
+	module RemoveFrontMatterFilter
+		def removefrontmatter input
+			# find the first
+			first = input.index("---")
+			if (first == nil)
+				input
+			end
+
+			# find the second
+			second = input.index("---", first + 1)
+			if (second == nil)
+				input
+			end
+
+			#strip the string
+			input[first..second + 2] = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.